src/index.html

```diff
@@ -20,8 +21,330 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
 
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a href="index.html">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="header__navigation">
+        <ul class="header__list">
+          <li class="header__link"><a class="is-active">Apple</a></li>
+          <li class="header__link"><a>Samsung</a></li>
+          <li class="header__link"><a>Smartphones</a></li>
+          <li
+            class="header__link"
+            data-qa="nav-hover"
+          >
+            <a>Laptops & Computers</a>
+          </li>
+          <li class="header__link"><a>Gadgets</a></li>
+          <li class="header__link"><a>Tablets</a></li>
+          <li class="header__link"><a>Photo</a></li>
+          <li class="header__link"><a>Video</a></li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+    <main class="main">
+      <!-- #region card -->
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="/src/images/imac.jpeg"
+          alt="Product"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__subtitle">Product code: 195434</p>
+
+        <div class="card__response">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__text">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <p class="left">Price:</p>
+          <p class="right">$2,199</p>
+        </div>
+        <a
+          href="#"
```

Comment:
The href attribute is set to '#', which is a placeholder and does not lead to a valid URL or action. Consider replacing it with a valid link or implementing a JavaScript function to handle the click event.